extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -218,10 +218,12 @@ class ExtractionPipelineBuilder(getClient: () => EncodeClient)
   ): SCollection[String] => SCollection[Msg] = { idStream =>
     val paramsBatchStream =
       idStream
-        .map(KV.of("key", _))
+        .withName("Apply fake keys")
+        .map(KV.of("", _))
         .setCoder(KvCoder.of(StringUtf8Coder.of(), StringUtf8Coder.of()))
         .applyKvTransform(GroupIntoBatches.ofSize(batchSize))
-        .map(_.getValue)
+        .withName("Strip away fake keys")
```

Comment:
The problem here was that multiple `map` calls all nest within the same scope, resulting in non-unique generated names. Fixed by giving names to 2 of the 3 (which double as comments for why we're doing what we're doing)

Reply:
I was trying to read more about the withName function and found this definition:
Return a Value from this Enumeration whose name matches the argument s.
Is that how it's being used here? Do we apply those strings "Apply fake keys" and "Strip away fake keys" elsewhere so that they return when these calls are made?

Reply:
That's a different withName. This one sets the name of the next transformation step in the pipeline (vs. using an auto-generated name).